model/registry/user.yaml

```
@@ -28,8 +28,15 @@ groups:
         type: string
         stability: experimental
         brief: >
-          Unique identifier of the user.
+          Identifier of a user who interacts with a system. This identifier may be unique only through best-effort means and does not imply that the user is authenticated to the system.
```

suggesting some clarifications:
```suggestion
          Identifies a user interactive with a system regardless of user authentication status. This identifier may be unique only through best-effort means.
        note: > 
          The `user.id`, when populated, is expected to be generated before user is authenticated and SHOULD NOT change after the user logs in. In browser scenarios `user.id` is usually stored in cookies.
          It's NOT RECOMMENDED to populate this attribute when unauthenticated users are not tracked or identified by the application.
```